src/Symfony/Component/Security/Guard/Provider/GuardAuthenticationProvider.php

```diff
@@ -56,7 +56,7 @@ public function __construct($guardAuthenticators, UserProviderInterface $userPro
     /**
      * Finds the correct authenticator for the token and calls it.
      *
-     * @param GuardTokenInterface $token
+     * @param TokenInterface|GuardTokenInterface $token
```

Comment:
I reverted this change as it should have been `TokenInterface` from day 1. I'm going to fix it on 2.7